packages/backend/constants/common.ts

```diff
@@ -71,6 +82,11 @@ export enum ChatStandardObjects {
     message = 'message',
 }
 
+export enum TicketStandardObjects {
+    ticketUser = 'ticketUser',
```

Comment:
Lets rename this to `user` and `task` itself since the enum is under `TicketStandardObject` it's meaning is implied.